concepts/Common/TypeScript Guides/90 Syntax Elements/90 Conditional Types.md

```diff
@@ -0,0 +1,7 @@
+When navigating through DevExtreme sources, you can find conditional types like the following:
+
+    export type DxEvent<TNativeEvent = Event> = {} extends EventType ? (EventObject & TNativeEvent) : EventType;
+
+    export type DxElement<T extends Element = HTMLElement> = {} extends Condition ? T : ElementWrapper<T>;
+
+These conditional types are needed for integrating with third-party libraries. You can ignore them. They resolve to built-in BrowserTypes: specific `HTMLElement` and browser `Event` extended with DevExtreme’s `EventObject` mixin.
```

Comment:
Not sure if "BrowserTypes" is actually name that should be used without a space (couldn't find that in Google). 

```suggestion
These conditional types enable integration with third-party libraries. You can ignore them. They resolve to built-in browser types: a specific `HTMLElement` or a browser `Event` extended with DevExtreme’s `EventObject` mixin.
```